### PR TITLE
chore: add volunteer JohnSaidi

### PR DESCRIPTION
**Pull request by JohnSaidi**
Adding _JohnSd as a New tech twitter volunteers.

### CODE REVIEW

File: data/users/JohnSaidi.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "<John SD>",
```
Comment (codewithvoid) [inline data/users/JohnSaidi.json:2]: remove < >. e.g
```json
"name": "John Saidi"
```

File: data/users/JohnSaidi.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "bio": "Full Stack Developer",
+  "image": "https://avatars.githubusercontent.com/u/60397319?v=4",
+  "categories": [
+    {
+      "slug": "programming | software-development",
```
Comment (codewithvoid) [inline data/users/JohnSaidi.json:7]: just mention programming here. e.g
```json
"slug": "programming"
```

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks @JohnSaidi for volunteering with AskDevs.

Added couple small comments. PTAL.

File: data/users/JohnSaidi.json (hunk 2 above)
Comment (JohnSaidi) [reply data/users/JohnSaidi.json]: Fixed

**Pull request merged**